tests/fr/homeassistant_HassTimerStatus.yaml

```diff
@@ -54,6 +54,14 @@ tests:
       - "Il reste combien de temps au minuteur appelé chocolatine"
       - "Il reste combien sur mon minuteur appelé chocolatine"
       - "Il reste combien de temps dans le minuteur appelé chocolatine"
+      - "Combien de temps reste-t-il pour la chocolatine"
+      - "Il reste combien de temps pour la chocolatine"
+      - "Combien de temps pour la chocolatine"
+      - "Il reste encore combien de temps pour la chocolatine"
+      - "Encore combien de temps reste-t-il pour la chocolatine"
+      - "Encore combien de temps pour la chocolatine"
+      - "Encore combien pour la chocolatine"
+      - "Combien pour la chocolatine"
```

Comment:
Yeah this is what I was afraid of ☺️

`Combien pour la chocolatine` for me is a question about how much it cost in a shop 😆 This is why we need more literally chunks on the sentence